README.md

```diff
@@ -133,8 +133,6 @@ To run the development environment, you will need the following:
 
 To install `node` on your system follow [these instructions](https://github.com/joyent/node/wiki/Installation#installing-without-building).
```

Comment:
agreed.  strangely much better than the link in the wiki instructions.  https://nodejs.org/download/